.github/actions/build/action.yml

```diff
@@ -50,7 +50,7 @@ runs:
       cache-from : type=gha,scope=smoo-build-env
       cache-to   : type=gha,scope=smoo-build-env,mode=max
   -
-    name  : Build mod
+    name  : Switch
```

Comment:
This step builds the mod for Switch AND for emulators. Naming it `Switch` is conceptually wrong.

Reply:
this change was only meant as a test on a local hidden repo, idk how it got here